README.md

```diff
@@ -39,8 +39,9 @@
 > [!IMPORTANT]  
 > This is the home for the shared theme format that [†
 > Vendetta](https://github.com/vendetta-mod/Vendetta),
-> [Enmity](https://github.com/enmity-mod/enmity) &
-> [Bunny](https://github.com/pyoncord/Bunny) use.
+> [Bunny](https://github.com/pyoncord/Bunny), [Revenge
+> ](https://github.com/revenge-mod/revenge-bundle) & [
+> Enmity](https://github.com/enmity-mod/enmity)  use.
 > Vendetta is no longer included in the instructions below, as it has reached
 > "End of Life."
```

Comment:
I'd make the entire notice:

```md
> [!IMPORTANT]
> Vendetta is no longer included in the instructions below, as it has reached
> "End of Life."
```

Mostly because we already know the list of ports it supports from the title.